### PR TITLE
Add dropped weight & volume to Multidrop menu stats

### PR DESCRIPTION
**Pull request by Zaraka**
#### Summary

SUMMARY: Interface "Add dropped weight & volume to Multidrop menu stats"

#### Purpose of change

When dropping items from inventory using Multidrop I have problems tracking the weight & volume of dropping items. Especially while working with vehicles.
Also generally I have problems tracking how much I'm actually dropping. The multidrop menu will show only how much the player will be carrying.

#### Describe the solution

After thinking about possible solutions I've decided that the multidrop menu could benefit from stats that would display how much weight & volume will be dropped when all of the selected items are dropped. 

Multidrop menu when no items are selected
Screenshot_20201212_222732

Multidrop menu after the player selects some items
Screenshot_20201212_222705

#### Describe alternatives you've considered

This information could be displayed in different parts of the window, but I personally think the stats header is the best choice.
In the future, this can be expanded to the Multidrop menu explaining where exactly will the items be dropped (floor, trunk, locker, e.t.c.)

#### Testing

Spawned into the game, tested dropping various items, tested dropping containers, tested different selection menu.

#### Additional context

This is my first C++ PR into this project, I've been literally in the source code for a few hours, so please pay extra attention if the things are done correctly.
Anyway, I've rewritten part of the inventory stats handling to resemble more a table approach.
Inventory windows stats windows now support a truly dynamic style of stats, stats are now strictly split into columns and every column has defined alignment, either left or right.
Stats assumes that both lines have the same size, but if requested I can do the extra steps to make this truly dynamic.

### CODE REVIEW

File: src/inventory_ui.cpp [hunk 1]
```diff
@@ -2623,9 +2640,43 @@ void inventory_drop_selector::set_chosen_count( inventory_entry &entry, size_t c
...
+
+std::pair<units::mass, units::volume> inventory_drop_selector::get_drop_weight_volume() const
+{
+    units::mass drop_mass = 0_gram;
+    units::volume drop_volume = 0_ml;
+    for( const std::pair<item_location, int> &loc_pair : dropping ) {
+        const item *dropped_item = loc_pair.first.get_item();
+        if( dropped_item ) {
+            drop_mass += dropped_item->weight() * loc_pair.second;
+            drop_volume += dropped_item->volume() * loc_pair.second;
```
Comment (mqrause) [inline src/inventory_ui.cpp:2677]: `item::volume` already accounts for charges, so for item stacks you multiply by charges twice here.

Comment (Zaraka) [reply src/inventory_ui.cpp]: Got it, I've tested stacks of simple items, loaded/unloaded magazines and single separate items, seems it works correctly now.

**Comment by 8street** [comment]
Can you add a minus signs before dropped weight/volume? I think it would be more logical.

**Comment by Zaraka** [comment]
> Can you add a minus signs before dropped weight/volume? I think it would be more logical.

And so I did

I've resolved the conflict, I've had to restructure a bit more stuff because with the longest length/free volume added in #46001 it would be a 4 more columns.
Anyway, current version for multidrop is this

Screenshot_20201214_223252

**Comment by Night-Pryanik** [comment]
Could you please test the case with three-digit numbers for volume and/or weight? E.g. something like `100/200 -> 100`. To check the numbers don't overlap other text and/or borders.

**Comment by Zaraka** [comment]
> Could you please test the case with three-digit numbers for volume and/or weight? E.g. something like `100/200 -> 100`. To check the numbers don't overlap other text and/or borders.

Does this suffice?

Screenshot_20201215_114235

**Comment by Night-Pryanik** [comment]
Yes, definitely.

**Comment by Zaraka** [comment]
It seems that AppVeyor build fail is out of my reach

**Comment by kevingranade** [comment]
The Appveyor failure isn't anything you need to worry about, but I don't find the change to be at all readable.

At this point we're cramming quite a lot of values into that header, we might need to rethink how this whole menu is layed out.

**Comment by Night-Pryanik** [comment]
@Zaraka could you please resolve conflicts?

**Comment by Zaraka** [comment]
> @Zaraka could you please resolve conflicts?

No, I will actually close this as Kevin already said he wants to do it differently because it looks cluttered after he introduced the volume & length specific pockets out of nowhere.
To be completely honest I've lost interest in Cataclysm because of this as I thought inventory management is over-complicated and super confusing, feel free to scrap the code and do whatever with it.

**Pull request closed without merging**